### DISCUSSION TITLE
`render --to pdf` and `::: {.content-visible when-profile="MYPROFILE"}`: why is preceding empty line required

### DISCUSSION THREAD

**Discussion by r-cheologist** [quarto-dev/quarto-cli#4885] [Q&A]
Consider a `*.qml` file like this:

    Profile-dependent content follows:

    ::: {.content-visible when-profile="MYPROFILE"}
    Content requiring `--profile MYPROFILE` to be added to a `render` call
    ::::
    
This works just fine, but when omitting the empty line preceding  `::: {.content-visible`, the container is not recognized and printed verbatim.

Is that by design or a bug?

**Comment by cscheid** [comment]
That's by design. What are you trying to accomplish? If you're looking for an in-paragraph version of content-visible, that's currently not supported: only entire blocks can be hidden or visible at once.

**Comment by mcanouil** [comment]
Duplicate of #4886
